### PR TITLE
ticdc: add ddl detail to bidirectional replication doc

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-bidirectional-replication.md [hunk 1]
```diff
@@ -38,16 +38,33 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
 
 ## 执行 DDL
 
-双向复制的集群不支持同步 DDL。
-
-如果需要执行 DDL，采取以下步骤：
-
-1. 暂停所有集群中需要执行 DDL 的对应的表的写入操作。如果是添加非唯一索引，不用暂停写入。
+开启双向复制功能后，TiCDC 不会同步任何的 DDL 。用户需要手动在开启双向复制的集群中执行 DDL。并且，因为有的 DDL 会造成 表结构的变更，可能会影响 TiCDC 同步数据的正确性。因此，在双向同步期间，只能不停机执行某些 DDL，下列表格给出了这些 DDL 和其注意事项。
```
Comment (nongfushanquan) [inline ticdc/ticdc-bidirectional-replication.md]: 用户需要自行在上下游集群中分别执行 DDL。另外需要注意的是，由于某些 DDL 会造成表结构变更或者数据更改时序的问题，会导致数据同步后出现不一致性问题，因此，在开启双向同步功能后，仅限下表中的 DDL 可以在不停业务数据写入的情况下执行。

File: ticdc/ticdc-filter.md [hunk 2]
```diff
@@ -88,3 +88,40 @@ ignore-update-new-value-expr = "gender = 'male' and age > 18" # 过滤掉新值
 > - TiDB 在更新聚簇索引的列值时，会将一个 UPDATE 事件拆分成为 DELETE 和 INSERT 事件，TiCDC 无法将该类事件识别为 UPDATE 事件，因此无法正确地进行过滤。
 >
 > - 在配置 SQL 表达式时，请确保符合 matcher 规则的所有表均包含了对应 SQL 表达式中指明的所有列，否则同步任务将无法创建成功。此外，若在同步的过程中表的结构发生变化，不再包含 SQL 表达式中的列，那么同步任务将会进入无法自动恢复的错误状态，你需要手动修改配置并进行恢复操作。
+
+## DDL 白名单
+
+目前 TiCDC 在同步 DDL 时使用白名单策略，只有在白名单内部的 DDL 才支持同步。不在白名单内的 DDL 会被 TiCDC 直接丢弃。
```
Comment (nongfushanquan) [inline ticdc/ticdc-filter.md:94]: 目前 TiCDC 在同步 DDL 时使用白名单策略，只有在白名单内部的 DDL 会同步到下游，不在白名单内的 DDL 会被 TiCDC 不会同步到下游。

File: ticdc/ticdc-bidirectional-replication.md [hunk 3]
```diff
@@ -38,16 +38,33 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
 
 ## 执行 DDL
 
-双向复制的集群不支持同步 DDL。
-
-如果需要执行 DDL，采取以下步骤：
-
-1. 暂停所有集群中需要执行 DDL 的对应的表的写入操作。如果是添加非唯一索引，不用暂停写入。
+开启双向复制功能后，TiCDC 不会同步任何的 DDL。用户需要自行在上下游集群中分别执行 DDL。另外需要注意的是，由于某些 DDL 会造成表结构变更或者数据更改时序的问题，会导致数据同步后出现不一致问题，因此，在开启双向同步功能后，仅限下表中的 DDL 可以在不停业务数据写入的情况下执行。
```
Comment (ran-huang) [inline ticdc/ticdc-bidirectional-replication.md:41]:
```suggestion
开启双向复制功能后，TiCDC 不会同步任何 DDL。用户需要自行在上下游集群中分别执行 DDL。

需要注意的是，某些 DDL 会造成表结构变更或者数据更改时序问题，从而导致数据同步后出现不一致的情况。因此，在开启双向同步功能后，只有下表中的 DDL 可以在业务不停止数据写入的情况下执行。
```

File: ticdc/ticdc-bidirectional-replication.md [hunk 4]
```diff
@@ -38,16 +38,33 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
...
+
+| Event                        | 是否会引起 changefeed 错误 | 说明    |
```
Comment (ran-huang) [inline ticdc/ticdc-bidirectional-replication.md:43]:
```suggestion
| 事件 | 是否会引起 changefeed 错误 | 说明 |
```

File: ticdc/ticdc-bidirectional-replication.md [hunk 5]
```diff
@@ -38,16 +38,33 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
...
+| ---------------------------- | ------ |--------------------------|
+| create database              | 是     | 错误可以自动恢复|
+| drop database                | 是     | 需要手动重启 changefeed，指定 `--overwrite-checkpoint-ts` 为该条 ddl 的commiTs 来恢复         |
```
Comment (ran-huang) [inline ticdc/ticdc-bidirectional-replication.md:46]:
```suggestion
| drop database                | 是     | 需要手动重启 changefeed，指定 `--overwrite-checkpoint-ts` 为该条 DDL 的 commiTs 来恢复         |
```

File: ticdc/ticdc-bidirectional-replication.md [hunk 6]
```diff
@@ -38,16 +38,33 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
...
+| create table                 | 是   | 错误可以自动恢复       |
+| drop table                   | 是   | 需要手动重启 changefeed，指定 `--overwrite-checkpoint-ts` 为该条 ddl 的commiTs 来恢复        |
```
Comment (ran-huang) [inline ticdc/ticdc-bidirectional-replication.md:48]:
```suggestion
| drop table                   | 是   | 需要手动重启 changefeed，指定 `--overwrite-checkpoint-ts` 为该条 DDL 的 commiTs 来恢复        |
```

File: ticdc/ticdc-bidirectional-replication.md [hunk 7]
```diff
@@ -38,16 +38,33 @@ TiCDC 复制功能只会将指定时间点之后的增量变更复制到下游
...
 
-双向复制的集群不支持同步 DDL。
-
-如果需要执行 DDL，采取以下步骤：
-
-1. 暂停所有集群中需要执行 DDL 的对应的表的写入操作。如果是添加非唯一索引，不用暂停写入。
+开启双向复制功能后，TiCDC 不会同步任何的 DDL。用户需要自行在上下游集群中分别执行 DDL。另外需要注意的是，由于某些 DDL 会造成表结构变更或者数据更改时序的问题，会导致数据同步后出现不一致问题，因此，在开启双向同步功能后，仅限下表中的 DDL 可以在不停业务数据写入的情况下执行。
+
+| Event                        | 是否会引起 changefeed 错误 | 说明    |
+| ---------------------------- | ------ |--------------------------|
```
Comment (ran-huang) [inline ticdc/ticdc-bidirectional-replication.md:44]: 这个表格里的事件是按一定顺序来排序的吗？能否改成按字母顺序排列

File: ticdc/ticdc-filter.md [hunk 8]
```diff
@@ -88,3 +88,40 @@ ignore-update-new-value-expr = "gender = 'male' and age > 18" # 过滤掉新值
 > - TiDB 在更新聚簇索引的列值时，会将一个 UPDATE 事件拆分成为 DELETE 和 INSERT 事件，TiCDC 无法将该类事件识别为 UPDATE 事件，因此无法正确地进行过滤。
 >
 > - 在配置 SQL 表达式时，请确保符合 matcher 规则的所有表均包含了对应 SQL 表达式中指明的所有列，否则同步任务将无法创建成功。此外，若在同步的过程中表的结构发生变化，不再包含 SQL 表达式中的列，那么同步任务将会进入无法自动恢复的错误状态，你需要手动修改配置并进行恢复操作。
+
+## DDL 白名单
+
+目前 TiCDC 在同步 DDL 时使用白名单策略，只有在白名单内部的 DDL 会同步到下游，不在白名单内的 DDL 会被 TiCDC 不会同步到下游。
```
Comment (ran-huang) [inline ticdc/ticdc-filter.md:94]:
```suggestion
目前 TiCDC 在同步 DDL 时使用白名单策略，只有在白名单内部的 DDL 会同步到下游，不在白名单内的 DDL 不会被 TiCDC 同步到下游。
```

File: ticdc/ticdc-filter.md [hunk 9]
```diff
@@ -88,3 +88,40 @@ ignore-update-new-value-expr = "gender = 'male' and age > 18" # 过滤掉新值
...
+
+以下为 TiCDC 支持同步的 DDL 的列表。
+
+1. create database
```
Comment (ran-huang) [inline ticdc/ticdc-filter.md:98]: 1. 请使用无序列表
2. 建议按字母顺序排列

File: ticdc/ticdc-bidirectional-replication.md (hunk 7 above)
Comment (asddongmen) [reply ticdc/ticdc-bidirectional-replication.md:44]: 可以的，没有顺序要求。

File: ticdc/ticdc-filter.md (hunk 9 above)
Comment (asddongmen) [reply ticdc/ticdc-filter.md:98]: 好的

Comment (asddongmen) [reply ticdc/ticdc-filter.md:98]: 现在是按照类别来排序，类别近似的排在一起，感觉也比较合理？

**Review by ran-huang — APPROVED** [review]

**Pull request merged**